### ISSUE TITLE
Multi-digit indices for listed formats prevent their selection

### ISSUE THREAD

**Issue by johnwait** [johnwait/youtube-dl_wrapper_script#6]
For downloads having more than 9 suitable streams, starting from the 10th one numerical indices are being used instead of letters as it should; e.g. instead of
```
    9) 360p  (mp4 @ 481k/s+audio; res: 480x360; size: 12.78MiB+audio)
    10) 480p  (mp4 @ 893k/s+audio; res: 640x480; size: 23.75MiB+audio)
  [Audio only]
    11) tiny  (m4a @ 128k/s+video; size: 4.73MiB+video)
```
the following should be displayed:
```
    9) 360p  (mp4 @ 481k/s+audio; res: 480x360; size: 12.78MiB+audio)
    A) 480p  (mp4 @ 893k/s+audio; res: 640x480; size: 23.75MiB+audio)
  [Audio only]
    B) tiny  (m4a @ 128k/s+video; size: 4.73MiB+video)
```
preventing the user from selecting them since the console at that point only expects a single keypress.

**Comment by johnwait** [comment]
Commit d4df3ad closes #6 as fixed.

**Comment by johnwait** [comment]
Second try: d4df3ad fixes #6
